docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -223,77 +228,123 @@ draws from Bayesian-derived joint posterior distributions.
+- Discrete time simulation can produce results that are statistically equivalent to those generated by discrete event simulation, assuming appropriate handling of time steps and event probabilities. 
+- Can be simpler to implement for scenarios where time steps align closely with the natural progression of events or states.
+
 
-  - Advantages: can use rates to determine varying amounts of time in different states (e.g. in rainy for longer than in cloudy, due to statistical probability). More computationally efficient
-  - Disadvantages: Need to know the maximum rate (or, if you have multiple states that you could go to, the sum of all the rates) in order to get a valid probability distribution In non-sink states (e.g., mortality is a sink state)
+There are, however, some disadvantages associated with discrete time simulation, such as: 
+
+- All simulants stay in each state for the same amount of time, which may not accurately reflect the variability seen in real-world scenarios.
+- Can be less computationally efficient compared to discrete event simulation, especially for large-scale models, because it requires evaluating the state of all simulants at each time step, regardless of whether events occur. 
+
+
+To read more about how we select the length of time steps in our simulations, please see `<vivarium_best_practices_time_steps>`_.
+
+
+Discrete event simulation 
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the other hand, discrete event simulation allows simulants to move from event to event, with the timing of these events determined by sampling from probability distributions. For this reason,
+discrete event simulation is synonymous with continuous time simulation; and could also be thought of as using a global clock. For an example of a microsimulation that uses discrete event simulation, 
+see the `Starsim series <https://www.ncbi.nlm.nih.gov/books/NBK293948/#:~:text=Discrete%20event%20simulation%20(DES)%20is,life%20process%2C%20facility%20or%20system>`_. As with discrete time simulation,
+there are some advantages and disadvantages associated with this type of event simulation.
+
+Advantages include: 
+
+- Can more accurately model the variability in time that simulants spend in different states, as the timing of transiitons is based on sampled events rather than fixed intervals. 
+- Can be more computationally efficient for many scenarios, as computations are only performed when events occur, rather than at every time step. This efficiency can be particularly significant in models where events are infrequent relative to the chosen time step in a discrete time simulation. 
+
+
+Disadvantages include:
+
+- Requires knowledge of the maximum rate or the sum of all rates leading out of a state to calculate valid transition probability. This can complicate model setup, especially in models with complex state transition dynamics.
```

Comment:
I'm not sure I completely follow this -- did it come from Starsim?

I think I might say:

```suggestion
- Requires knowledge of how to sample event times, as opposed to event probabilities. This can complicate model setup, especially in models with complex state transition dynamics.
```